### DISCUSSION TITLE
Docs can be translate chinese?

### DISCUSSION THREAD

**Discussion by chenguzhen87** [denoland/deno#8853] [Ideas]
docs  can be translate chinese?

**Comment by kitsonk** [comment]
There are quite a few resources in Chinese: https://github.com/denolib/awesome-deno#chinese

Specifically there is a discussion board: https://deno.js.cn/

And a community translation of the manual, where contributions are welcome: https://github.com/Nugine/deno-manual-cn/


中文资源很多：https://github.com/denolib/awesome-deno#chinese

具体来说，有一个讨论区：https://deno.js.cn/

以及该手册的社区翻译，欢迎您提供帮助：https://github.com/Nugine/deno-manual-cn/
